tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/ServiceRecipients Put.feature

```diff
@@ -0,0 +1,120 @@
+﻿Feature: Update the Service Recipients for an order
```

Comment:
Should a test be added to check the last updated by name and/or ID is updated when sending an update to the service recipients list?